src/org/opendatakit/briefcase/ui/export/components/FormsTableViewModel.java

```diff
@@ -128,7 +129,8 @@ private void updateDetailButton(FormStatus form, JButton button) {
   }
 
   private void updateConfButton(FormStatus form, JButton button) {
-    button.setForeground(forms.hasConfiguration(form) ? NO_CONF_OVERRIDE_COLOR : DARK_GRAY);
```

Comment:
This is getting outside of the ExportConfiguration stuff that we shouldn't have to care about here. I'd like to maintain the restriction of having to ask for `forms.hasConfiguration(form)`, not to see if `forms.getCustomConfiguration(form).map(f -> !f.isUserConfigsEmpty()).orElse(false)`.

This comment also affects the rest of changes that you did to support this:
- The different definitions of emptiness of an ExportConfiguration `isUserConfigsEmpty()` vs. `isEmpty()`
- The change on the behavior of the `removeConfiguration(FormStatus form)` method